### PR TITLE
NuRV FMU Monitor Service example

### PR DESCRIPTION
**Pull request by mortenhaahr**
A new example where the NuRV FMU is reused into a service.

### CODE REVIEW

File: digital_twins/incubator-NuRV-fmu-monitor-service/README.md
Comment (prasadtalasila) [inline digital_twins/incubator-NuRV-fmu-monitor-service/README.md:1]: The tutorial paper has a figure and matching text for this example. Can they be brought here and a reference provided to them?
The same observation also holds true for all the NuRV based DTs.

**Review by prasadtalasila — COMMENTED** [review]
@mortenhaahr Thanks for the new example. A couple of comments.

1. Are there any possible refactoring opportunities to reuse code between **Incubator NuRV Monitor Service** and **Incubator NuRV FMU Monitor Service** examples?
2. This DT example has model and possibly services put inside `digital_twins/incubator-NuRV-fmu-monitor-service`. Is it possible to move them to **model** and **service** (new directory) directories?
3. Like the comment in PR #51 , the lifecycle scripts for this example seem significantly different from those of plain incubator example.

**Comment by mortenhaahr** [comment]
1. It is possible to reuse some code in execute.py between them but I am not sure that I see the benefit of this.
Since it is simply "scenario code" I find it more readable to have the full scenario code inside each execute.py file instead of splitting it into separate files. 
This also leaves the two examples decoupled in case we want to change an example.
This way one does not need to go back and forth between files. What do you think?

2. Sure. This was copied from the NuRV-monitor-validation example. How about I create a folder in <DTaaS-examples>/models/incubator and put them there?

3. The plain incubator lifecycle scripts simply start the Incubator and do not demonstrate a specific scenario. One can interact with it as one sees fit (as described in common/digital_twins/incubator/README.md). The RV examples run a scenario for demonstrative purposes.

4. (Regarding figures). Good idea. But perhaps we should do this in another branch that is not specific to the NuRV Monitor Service example

**Comment by clagms** [comment]
@mortenhaahr you can find some code to run scenarios in the integration tests of the incubator... but these are quite brittle at the moment: https://github.com/INTO-CPS-Association/example_digital-twin_incubator/blob/master/software/integration_tests/dt_integration_test.py

**Comment by mortenhaahr** [comment]
Thank you for the input @clagms! I can refer the tutorial audience to this if they want to see more Incubator functionality.

@prasadtalasila I have now pushed 2. and 4. from above. Would you please have another look to see if this can be merged?

**Comment by prasadtalasila** [comment]
@mortenhaahr thanks for the PR.

**Pull request merged**